web-common/src/features/connectors/explorer/TableMenuItems.svelte

```diff
@@ -79,7 +107,14 @@
   </NavigationMenuItem>
 {/if}
 
-{#if showGenerateMetricsAndDashboard}
+{#if isYamlModelingSupported}
```

Comment:
Here's a case where this variable name is confusing. When modeling is supported, both SQL and YAML are supported.